src/components/mgt-person/mgt-person.ts

```diff
@@ -33,7 +33,6 @@ import { findUserByEmail } from './mgt-person.graph';
  * @cssprop --avatar-font-size - {Length} Avatar font-size when both name and email are shown
  * @cssprop --avatar-border - {String} Avatar border
  * @cssprop --initials-color - {Color} Initials color
- * @cssprop --initials-background-color - {Color} Initials background color
```

Comment:
This is a breaking change.

You should keep this css custom property and remove the `personAvatarBg`. See other comment

Reply:
Didn't we say we wanted the colors as part of the properties so that it will always be available after loading?

Reply:
In regards to removing `personAvatarBg`

Reply:
Yeah, you should have it as a private property

Reply:
Updated to `_personAvatarBg` to follow new pattern